react/TaxInfo.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FunctionComponent } from 'react'
+import { Tooltip, IconInfo } from 'vtex.styleguide'
+import TranslateTotalizer from 'vtex.totalizer-translator/TranslateTotalizer'
+
+import FormattedPrice from './components/FormattedPrice'
+
+const TaxInfo: FunctionComponent<Props> = ({ taxesTotals, className = '' }) => {
+  const tooltipLabel = (
+    <ul className="list pa0 ma0">
+      {taxesTotals.map(total => (
+        <li className="flex justify-between items-center" key={total.name}>
+          <span className="pr5">
+            <TranslateTotalizer totalizer={total} />
+          </span>
+          <span>
+            <FormattedPrice value={total.value} />
+          </span>
+        </li>
+      ))}
+    </ul>
+  )
+
+  return (
+    <span className={className}>
```

Comment:
That might not be the way. You can see @kaisermann 's comment about margin

Reply:
I like the idea of putting the margin outside. It rarely makes sense for a component to modify the spacing to its surrounding elements

Reply:
let's agree to agree

Reply:
I agree with you agreeing with me agreeing with you again.

Reply:
I agree with both